client/src/utils/api.js

```diff
@@ -0,0 +1,11 @@
+export async function getGroceriesHeader() {
+  const result = await fetch(`http://localhost:3001/api/groceries`);
+  const groceriesHeader = result.json();
+  return groceriesHeader;
+}
```

Comment:
over all , why there are two different functions, doing exact the same thing?

```suggestion
```

Reply:
This is a preparation for further steps. Function is not called yet.

Reply:
alright, maybe leave a comment for that, so other who review your code, will know